### PR TITLE
Fix render onresize

### PR DESCRIPTION
**Pull request by hrgdavor**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Does your submission pass tests?

render needs to be called after window resize

### CODE REVIEW

File: packages/web/src/ui/views/viewer.js [hunk 1]
```diff
@@ -58,6 +58,8 @@ const viewer = (state, i18n) => {
     render = prepareRender(viewerOptions)
     const gestures = pointerGestures(el)
 
+    if(typeof window != 'undefined') window.addEventListener('resize', evt=>updateView = true)
```
Comment (z3dev) [inline packages/web/src/ui/views/viewer.js:61]: Please adjust to follow syntax rules for spacing, functions, etc.

**Review by z3dev — CHANGES_REQUESTED** [review]
Small change requested otherwise this is fine.

File: packages/web/src/ui/views/viewer.js (hunk 1 above)
Comment (hrgdavor) [reply packages/web/src/ui/views/viewer.js]: ok, forgot about that :) .. I have applied standardx to the file, also removed `if(typeof window != 'undefined')` as window global is used in few more places.

**Review by z3dev — APPROVED** [review]
Super. Thanks.

**Pull request merged**